sandbox/extension/distribution_2021-02-15.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xsi:schemaLocation='http://rs.gbif.org/extension/ https://raw.githubusercontent.com/gbif/rs.gbif.org/master/schema/extension_2021-02-15.xsd'
+    rowType='http://rs.gbif.org/terms/1.0/Distribution'
+    dc:title='Species Distribution'
+    name='Distribution'
+    namespace='http://rs.gbif.org/terms/1.0'
+    dc:issued='2021-02-15'
+    dc:subject='dwc:Taxon'
+    dc:relation='http://rs.gbif.org/extension/gbif/1.0/distribution.xml'
```

Comment:
should this not point to somewhere else than the first version of the extension? Its a link to further documentation, which has been used to point to the GBIF hosted version. Guess this should be http://rs.gbif.org/extension/gbif/1.0/distribution_2021-02-15.xml ?

Reply:
Yes.

Reply:
@tucotuco can you update this to `http://rs.gbif.org/extension/gbif/1.0/distribution_2021-02-15.xml` so this discussion can be resolved?